File: Day 19/math.py
import sys
input = sys.stdin.readline

t = int(input())
for _ in range(t):
    a, b, c = map(int, input().split())
    x = sorted([a, b, c])
    if x[2] > x[0] + x[1]:
        x[2] = x[0] + x[1]
    print(x[2] - x[0])

File: Day 24/math.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let lines = [];
rl.on('line', (line) => {
  lines.push(line.trim());
}).on('close', () => {
  const t = parseInt(lines[0], 10);
  for (let i = 1; i <= t; i++) {
    const n = parseInt(lines[i], 10);
    if (n % 2 === 1) {
      console.log(0);
    } else {
      console.log(Math.floor(n / 4) + 1);
    }
  }
});
